packages/core/middleware/package.json

```diff
@@ -1,17 +1,21 @@
 {
   "name": "@vue-storefront/middleware",
-  "version": "2.1.1-rc.1",
+  "version": "2.1.1",
   "description": "",
-  "main": "module.js",
+  "main": "lib/index.cjs.js",
+  "module": "lib/index.es.js",
+  "types": "lib/src/index.d.ts",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "rimraf lib && rollup -c"
   },
   "author": "Vue Storefront",
   "license": "MIT",
   "dependencies": {
+    "consola": "2.15.3",
     "express": "^4.17.1",
     "cookie-parser": "^1.4.5",
-    "is-https": "^3.0.2"
+    "is-https": "^3.0.2",
```

Comment:
I see no usage of `is-https` in `core/middleware`

Reply:
used in _proxyUtils

Reply:
But `_proxyUtils` is inside `core/core` which has own `package.json`. Not inside `core/middleware`